frontend/src/App.res

```diff
@@ -20,8 +20,9 @@ let makeGetBenchmarksVariables = (
   ~startDate,
   ~endDate,
 ): GetBenchmarks.t_variables => {
+  open AppHelpers
   let isMaster = Belt.Option.isNone(pullNumber)
-  let isDefaultBenchmark = Belt.Option.isNone(benchmarkName)
+  let benchmarkName = Belt.Option.isNone(benchmarkName) ? Some(defaultBenchmarkName) : benchmarkName
```

Comment:
Is `benchmarkName` always `Some`? (can we remove the option, or is there still a usecase for `None`?)

Reply:
Should we do this in a separate PR, since we'd like to redeploy this quickly?

Reply:
I also pushed a commit that changes the graphql schema to correctly mark the `benchmark_name` field as non-NULL which we changed in a recent fix/migration. Also, removed the unnecessary `option` for `benchmarkName`

Reply:
Updating the graphql_schema actually breaks a bunch of other type checks. Looking into it. :)

Reply:
Okay, this should be ready for another look, hopefully to be merged. :) Thanks for your patience, @art-w !